pkg/pillar/cmd/nim/nim.go

```diff
@@ -597,7 +584,7 @@ func Run(ps *pubsub.PubSub, loggerArg *logrus.Logger, logArg *base.LogObject) in
 			start := time.Now()
 			if !ok {
 				log.Infof("Network testBetterTimer stopped?")
-			} else if dnc.NextDPCIndex == 0 {
+			} else if dnc.NextDPCIndex == 0 && !dnc.DeviceNetworkStatus.HasErrors() {
```

Comment:
The periodic try should clear the error already. Is that not the case?

Reply:
If the error happens because of port not being available, we do not start dhcpcd on such ports during periodic connectivity tests. We should launch verification process in order to go through the port list again and take care of dhcp.

Reply:
Define "not being availble". That the interface does not exist in the Linux kernel? Or something else?

Reply:
I meant the interface not being present in kernel.

Reply:
OK